### ISSUE TITLE
kworker : kacpi_notify using 90% CPU

### ISSUE THREAD

**Issue by Desdra** [jakeday/linux-surface#318]
Hi,

I just installed linux on my SB2 and I noticed that the fans were always spinning. I checked top and saw that I had a process that was taking 90% of a core constantly surely causing this overheating...

here is what top displays me : 
 ```299 root      20   0       0      0      0 R  91.7   0.0   3:15.12 kworker/0:2+kacpi_notify                           
  366 root      20   0       0      0      0 D  14.0   0.0   0:13.13 kworker/u16:4+kacpi_hotplug                        
  141 root      20   0       0      0      0 I   5.3   0.0   0:08.08 kworker/u16:2-events_unbound                       
  639 soudini   20   0 2985024 116920  79268 S   1.3   0.7   0:06.37 kwin_x11                                           
  445 root      20   0  287688  85352  46048 S   1.0   0.5   0:06.05 Xorg                                               
  703 soudini   20   0  452628  71996  57436 S   1.0   0.4   0:01.53 konsole                                            
    5 root      20   0       0      0      0 I   0.7   0.0   0:13.11 kworker/0:0-rcu_gp                                 
   86 root      20   0       0      0      0 I   0.3   0.0   0:14.22 kworker/u16:1-events_unbound
```

I installed your kernel through the steps described in the README and I am running Arch linux with KDE what can I do ? 

As far as I have tried it is impossible to kill the process and it is constantly running...

**Comment by qzed** [comment]
This shouldn't happen with a kernel containing the patches of this repo. Can you make sure you're booting the right one (the Arch kernel should be newer and is thus probably the default one)?

If you want some more details: The problem is the missing driver for the EC on which I'm currently working on, due to which the lid status check, which relies on the base/clipboard-status check constantly notifies itself (there is however a workaround in this repo which should avoid this spinning). The underlying issue will be fixed with #146.

**Comment by Desdra** [comment]
I am feeling kinda dumb right now... You were totally right I wasn't using your kernel... With it, everything works fine.... Thank you for your work !!!!
